Dockerfile.build

```diff
@@ -1,7 +1,11 @@
 ARG GO_VERSION
 FROM golang:${GO_VERSION}-buster
 
-RUN apt-get update && apt-get install -y zip
+RUN \
+  apt-get update \
+  && apt-get install -y zip \
+  && mkdir /.cache \
```

Comment:
What is this dir used by?

Reply:
The go-build tool seems to use it to cache some data.